lib/tether/toolbox.go

```diff
@@ -201,9 +212,17 @@ func (t *Toolbox) killHelper(session *SessionConfig, name string) error {
 
 	num := syscall.Signal(sig.Signum())
 
-	log.Infof("sending signal %s (%d) to process group for %s", sig.Signal, num, session.ID)
-	if err := syscall.Kill(-session.Cmd.Process.Pid, num); err != nil {
-		return fmt.Errorf("failed to signal %s group: %s", session.ID, err)
+	var pid int
+	if groupKill == false {
+		pid = session.Cmd.Process.Pid
+		log.Infof("sending signal %s (%d) to process %d for %s", sig.Signal, num, pid, session.ID)
+	} else {
+		pid = -session.Cmd.Process.Pid
+		log.Infof("sending signal %s (%d) to process group %d for %s", sig.Signal, num, -pid, session.ID)
```

Comment:
Is the `-pid` intended here - it's a double negative but maybe that's what you were after explicitly for printing the pgid?